media-gfx/hydrus/hydrus-445.ebuild

```diff
@@ -10,13 +10,21 @@ inherit python-single-r1 desktop optfeature
 
 DESCRIPTION="A booru-like media organizer for the desktop"
 HOMEPAGE="https://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
-SRC_URI="https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+if [ "${PV}" == "9999" ]; then
```

Comment:
Bash [tests](https://devmanual.gentoo.org/tools-reference/bash/#single-versus-double-brackets-in-bash)!

Reply:
Fixed for all versions, plus updated 9999.